common/src/main/java/org/opensearch/ml/common/conversation/ActionConstants.java

```diff
@@ -29,6 +29,8 @@ public class ActionConstants {
     public final static String RESPONSE_CONVERSATION_LIST_FIELD = "conversations";
     /** name of list on interactions in all responses */
     public final static String RESPONSE_INTERACTION_LIST_FIELD = "interactions";
+    /** name of list on traces in all responses */
```

Comment:
I think this memory related are already merged into main. Can you rebase this change against the main? I see quite a lot of the Memory code are duplicates in the PR.

Reply:
+1 on this